src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -546,7 +564,7 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_HandleCloseTabsAfter(const IInspectable& /*sender*/,
                                              const ActionEventArgs& actionArgs)
     {
-        if (const auto& realArgs = actionArgs.ActionArgs().try_as<CloseTabsAfterArgs>())
+        if (const auto realArgs = actionArgs.ActionArgs().try_as<CloseTabsAfterArgs>())

```

Comment:
I think you meant to do this for `_HandleCloseTab`, not `_HandleCloseTabsAfter` haha

Also, @lhecker, I'm curious, why should these be `const auto` instead of `const auto&`? Wouldn't it be better to reference the struct instead of copying it? Or am I misunderstanding what's going on.

Reply:
Whoops - reverted

Reply:
@carlos-zamora `as` and `try_as` return a temporary. You can't hold a reference to a temporary legally in the language.

The reference to the temporary at local scope is promoted to a copy.

Reply:
There are quite a few of these in this file FYI :)